postgres/test/test_postgres.py

```diff
@@ -214,28 +222,32 @@ def test_check(self):
 
         # instance connection metrics
         for mname in self.CONNECTION_METRICS:
-            self.assertMetric(mname, count=1)
+            self.assertMetric(mname, count=1, tags=[])            
+            self.assertMetric(mname, count=1, tags=['postgresinstance: additionalTag'])            
+
 
         # db level connections
         for inst in instances:
-            expected_tags = ['db:%s' % inst['dbname']]
+            expected_tags = ['db:%s' % inst['dbname'], instance_tag]
             self.assertMetric('postgresql.connections', count=1, tags=expected_tags)
 
         # By schema metrics
+        self.assertMetric('postgresql.table.count', value=1, count=1, tags=['schema:public', instance_tag])
         self.assertMetric('postgresql.table.count', value=2, count=1, tags=['schema:public'])
+
         self.assertMetric('postgresql.db.count', value=2, count=1)
 
         # Our custom metric
-        self.assertMetric('custom.numbackends', value=1, tags=['customdb:datadog_test'])
+        self.assertMetric('custom.numbackends', value=1, tags=['customdb:datadog_test', instance_tag])
 
         # Test service checks
         self.assertServiceCheck('postgres.can_connect',
             count=1, status=AgentCheck.OK,
-            tags=['host:localhost', 'port:15432', 'db:datadog_test']
+            tags=['host:localhost', 'port:15432', 'db:datadog_test', "tags:['postgresinstance: additionalTag', 'db:datadog_test']"]
```

Comment:
Tags from the instance should be at the same level as tags from the integration:

```python
tags=['host:localhost', 'port:15432', 'db:datadog_test', "postgresinstance:additionalTag", "db:datadog_test"]
```
It should be the same as for the metric's tags.

Same for the following changes.